airflow/providers/sftp/hooks/sftp.py

```diff
@@ -224,9 +224,7 @@ def retrieve_file(self, remote_full_path: str, local_full_path: str) -> None:
         :type local_full_path: str
         """
         conn = self.get_conn()
-        self.log.info('Retrieving file from FTP: %s', remote_full_path)
```

Comment:
Probably change to debug level instead of removing?

Reply:
@kaxil that's a way to solve it as well. I'm not really that opinionated about it as long as they don't get into INFO level.

Reply:
Paramiko already has debug level logging for this, so IMO we don’t need to duplicate here.

Reply:
Good point, yeah